### ISSUE TITLE
Clear system clipboard when clicking on clear button clipboards.

### ISSUE THREAD

**Issue by pokegamer5547** [florisboard/florisboard#811]
having a way to clear system clipboard when clicking on clear clipboard section of keyboard would be great since that button only clear from keyboard itself. (maybe this can be an optional and be enabled in settings.)

**Comment by pokegamer5547** [comment]
I also have a new idea, on clipboard button there's a lot of empty space left on options list. Maybe having an button there to only clear the system clipboard would be great. This way (along with android10 clipboard chnage making only the foreground app can access clipboard) will let others able to copy text(especially in websites) and clear it from system by using florisboard while still having the copied text in florisboard. Having it little more private way to use clipboard.
![IMG_20210514_132600](https://user-images.githubusercontent.com/65234485/118239906-f8bbbc80-b4b7-11eb-8d05-3b831917c313.jpg)

**Comment by Luensche** [comment]
I am wondering if the clipboard history is the right place for a button to delete the system clipboard. I understand the system clipboard as the further clipboard. So if I clean the system clipboard, I would expect the internal clipboard cleaned as well.

**Comment by pokegamer5547** [comment]
> I am wondering if the clipboard history is the right place for a button to delete the system clipboard. I understand the system clipboard as the further clipboard. So if I clean the system clipboard, I would expect the internal clipboard cleaned as well.

Yes you are also right it should probably clean it too but the reason i wanted this instead is because i trust to keep clipboard history with floris but not system clipboard(because any app can access it). So maybe it can work like maybe having an option in settings which will make it delete only from system.

**Comment by rancidfrog** [comment]
The delete button in main clipboard smartbar, seems more appropriate icon for clear clipboard history.

Also,
In regards to delete current clipboard item, what use cases does it apply to?
Is there a specific reason it is needed?
Thank you

**Comment by rancidfrog** [comment]
@pokegamer5547 
In that case, you might want to use the setting to sync clipboard with system:
![](https://i.imgur.com/77Na3Vk.jpg)

That way you always know that whatever is on floris clipboard is also on system clipboard.
Also, it will automatically clear system history if you clear it on app, if I am not mistaken as they are synced

**Comment by pokegamer5547** [comment]
@rancidfrog thanks for your explanation! I actually use buttons of florisboard to copy stuff on text fields. But its not possible to copy text in browser,etc using keyboard.
 What i want is to be able to clean those texts from system clipboard with keyboard app so we do not need to worry about what if another app is capturing it. I cannot use "sync to system clipboard"  as it will sync all texts i copy to florisboard which i dont want.

**Comment by patrickgold** [comment]
Above PR adds detailed sync behavior for both directions, allowing to differentiate between no events, only clear, only set, or all events. You can thus set syncToSystem to "Only clear events" and have your desired behavior.
